src/python/espressomd/visualization_opengl.py

```diff
@@ -32,8 +32,7 @@
 from .particle_data import ParticleHandle
 
 
-class openGLLive:
-
+class openGLLive(object):
```

Comment:
not needed

Reply:
True, as we are using Python 3 &ndash; followed the suggestion of pycharm not realizing that I still had it configured for Python 2, where explicit inheritance was necessary to prevent use of old-style classes. Removed it again.